.ci/release.yml

```
@@ -5,39 +5,50 @@ parameters:
   displayNameJob2: 'Release PSResourceGet to PSGallery'
 
 jobs:
-#- job: ${{ parameters.jobName1 }}
-#  pool:
-#    name: 1ES
-#    demands:
-#    - ImageOverride -equals PSMMS2019-Secure
-#  displayName: ${{ parameters.displayNameJob1 }}
+- job: ${{ parameters.jobName1 }}
+  pool:
+    name: 1ES
+    demands:
+    - ImageOverride -equals PSMMS2019-Secure
+  displayName: ${{ parameters.displayNameJob1 }}
 
-#  steps:
-#  - task: DownloadPipelineArtifact@2
-#    displayName: 'Download PSResourceGet module artifacts'
-#    inputs:
-#      artifact: nupkg
-#      patterns: '**/*.nupkg'
-#      downloadPath: '$(Pipeline.Workspace)/nuget'
+  steps:
+  - task: DownloadPipelineArtifact@2
+    displayName: 'Download PSResourceGet module artifacts'
+    inputs:
+      artifact: nupkg
+      patterns: '**/*.nupkg'
+      downloadPath: '$(Pipeline.Workspace)/nuget'
 
-  #- powershell: |
-  #    $package = (Get-ChildItem '$(Pipeline.Workspace)/nuget/Microsoft.PowerShell.PSResourceGet.*.nupkg').FullName
-  #    $package
-  #    $vstsCommandString = "vso[task.setvariable variable=NugetPkgPath]${package}"
-  #    Write-Host "sending " + $vstsCommandString
-  #    Write-Host "##$vstsCommandString"
-  #  displayName: 'Capture PSResourceGet module NuGet package path and set environment variable'
+  - powershell: |
+      $package = (Get-ChildItem '$(Pipeline.Workspace)/nuget/Microsoft.PowerShell.PSResourceGet.*.nupkg').FullName
+      $package
+      $vstsCommandString = "vso[task.setvariable variable=NugetPkgPath]${package}"
+      Write-Host "sending " + $vstsCommandString
+      Write-Host "##$vstsCommandString"
+    displayName: 'Capture PSResourceGet module NuGet package path and set environment variable'
+
+  - task: NuGetAuthenticate@1
+    inputs:
+      nuGetServiceConnections: PSCoreReleasePush
+
+  - task: ManualValidation@0
+    displayName: Apporove release to PS internal feed
```

nitpick/non-blocking
```suggestion
    displayName: Approve release to PS internal feed
```